README.md

```diff
@@ -0,0 +1,175 @@
+- **[Add a Game]**  
+- **[Add a Music Album]**
+- **[List all added items]**
+- **[Keep track of books, music albums, movies, and games and other information]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Project Documentation <a name="live-demo"></a>
+
+Please visit our zoom video presentation. Click the [Link](https://loom.com/share/cacf0349f8cb4db49d90f9b9323de431) button.
+
+
+### Diagram
+![Diagram](https://user-images.githubusercontent.com/35680940/211393165-1c878ea1-12e4-4f7f-a749-f3a3c7823c7e.png)
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+To get a local copy up and running follow these simple example steps.
+
+## Prerequisites
+Have `Ruby`, and `Git` installed on your local machine.
+
+### Setup
+
+Clone this repository to your desired folder:
+
+- git clone https://github.com/MarcoAurelioAntonio/Catalog
+- cd Catalog
+- Run `bundle install` to install the required gems
+
+
+## Usage
+Run `main.rb` file in the terminal:
+ruby `main.rb`
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## Author <a name="authors"></a>
+
+👤 **Camilo Vela**
+
+- GitHub: [@camilovelag](https://github.com/camilovelag)
+- Twitter: [@camilovelag](https://twitter.com/camilovelag)
+- LinkedIn: [camilovelag](https://linkedin.com/in/camilovelag)
+
+👤 **Solomon Akpuru**
+- GitHub: [@githubhandle](https://github.com/solobarine)
+- Twitter: [@twitterhandle](https://twitter.com/solomonakpuru)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/solomon-akpuru)
+
+👤 **Gustavo Porcilio**
+
+- GitHub: [@MarcoAurelioAntonio](https://github.com/MarcoAurelioAntonio)
+- Twitter: [@gustavoporcilio](https://twitter.com/gustavoporcilio)
+- LinkedIn: [Gustavo Porcilio](https://www.linkedin.com/in/gustavo-porcilio-4496a223a/)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- **[Add label and author validations as future improvements]**
+- **[Persist one-to-many relationships]**
+- **[Improve listing display]**
+- **[Improve UI using diferent colors and bold]**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/MarcoAurelioAntonio/Catalog/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project, please leave star
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FAQ (optional) -->
+
+## ❓ FAQ <a name="faq"></a>
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+

```

Comment:
- [ **OPTIONAL** ] **IMPORTANT:** What do you think to fill all your README sections with their expected content to make it more professional? Right now, most of the  selected above they appear to be empty

Reply:
Done! 😄